web/client/components/TOC/enhancers/tocItemsSettings.js

```diff
@@ -35,31 +34,7 @@ export const settingsState = compose(
  */
 export const settingsLifecycle = compose(
     withHandlers({
-        onClose: ({ onUpdateInitialSettings = () => {}, onUpdateOriginalSettings = () => {}, onUpdateNode = () => {}, originalSettings, settings, onHideSettings = () => {}, onShowAlertModal = () => {} }) => (forceClose, tabsCloseActions = []) => {
-            const originalOptions = Object.keys(settings.options).reduce((options, key) => ({ ...options, [key]: key === 'opacity' && !originalSettings[key] && 1.0 || originalSettings[key] }), {});
-            if (!isEqual(originalOptions, settings.options) && !forceClose) {
-                onShowAlertModal(true);
-            } else {
-                if (isArray(tabsCloseActions)) {
-                    tabsCloseActions.forEach(tabOnClose => {
-                        if (isFunction(tabOnClose)) {
-                            tabOnClose();
-                        }
-                    });
-                }
-                onUpdateNode(
-                    settings.node,
-                    settings.nodeType,
-                    { ...settings.options, ...originalSettings }
-                );
-                onHideSettings();
-                onShowAlertModal(false);
-                // clean up internal settings state
-                onUpdateOriginalSettings({});
-                onUpdateInitialSettings({});
-            }
-        },
-        onSave: ({ onUpdateInitialSettings = () => {}, onUpdateOriginalSettings = () => {}, onHideSettings = () => { }, onShowAlertModal = () => { } }) => (tabsCloseActions = []) => {
+        onClose: ({ onUpdateInitialSettings = () => {}, onHideSettings = () => {} }) => (forceClose, tabsCloseActions = []) => {
```

Comment:
```suggestion
        onClose: ({ onUpdateInitialSettings = () => {}, onHideSettings = () => {} }) => (tabsCloseActions = []) => {
```
I think `forceClose` is not required anymore. Also kindly make changes in TOCItemSettings `onClose` call and corresponding unit tests in `tocItemSettings`